docs/Developer-guide/Building-GlusterFS.md

```diff
@@ -67,7 +67,7 @@ The following yum command installs the build requirements for CentOS / Enterpris
     pyxattr readline-devel rpm-build sqlite-devel systemtap-sdt-devel \
     tar userspace-rcu-devel
 ```
-Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package
+Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package<br>
```

Comment:
This should be
```
**Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package**
```
Which will give the below output:
**Note: You will need to enable the CentOS SIG repos in order to install userspace-rcu-devel package**

Reply:
I have left the opportunity any reviewer to modify the code.
Anyway - formatting for me is not so important than the info that the user needs the CentOS SIG repos for some packages.